config/locales/en.yml

```diff
@@ -247,6 +247,26 @@ en:
       para1: "The following abuse report has been sent to the Abuse team:"
       url_para: "URL of the page the abuse is on:"
       description_para: "Description of the abuse:"
+    challenge_assignment_notification:
+      subject: "[%{app_name}] [%{collection_title}] Your Assignment!"
```

Comment:
You're right, the challenge ones don't - I got confused by the other user mailer ones that do.